packages/tizen_rpc_port/example/client/lib/message_client.dart

```diff
@@ -107,7 +107,7 @@ class Message extends ProxyBase {
       }
       return parcel;
     } catch (error) {
-      print(error.toString());
+      print(error);
```

Comment:
Can we remove the catch clause so that the parcel parsing error can be propagated to the user code (as an unhandled exception) properly?

Reply:
Ok, I'll prepare modified patch. :)

Reply:
https://review.tizen.org/gerrit/c/platform/core/appfw/tidl/+/287255